_info/Tribute

```diff
@@ -1,6 +1,6 @@
 Tribute
 
-During each day phase, the team's kings have to choose to tribute one of their team members, with a certain amount of money assigned to the tribute. This is mandatory. The lowest paying team will have their tribute killed. 
+During each day phase, each team's kings have to choose to tribute one of their team members with a certain amount of money assigned to the tribute. This is mandatory. The lowest paying team will have their tribute killed. 
```

Comment:
oh last thing, just noticed. the proper grammar would be "each teams king" instead of "each teams kings"

Reply:
alright